### PR TITLE
chore: Test Docusaurus webpack 5 support

### PR DESCRIPTION
**Pull request by slorber**
Use an unreleased version of Docusaurus to test the webpack5 support works for this site.

See also https://github.com/facebook/docusaurus/pull/4089

### CODE REVIEW

**Comment by slorber** [comment]
weird, netlify does not work 😓 (also on last commit of main)

**Comment by slorber** [comment]
@satya164 @eriveltonelias it looks like the deploy previews are broken on Netlify.

Can you click on "retry / clear cache" here please? I think it may solve the problem.

https://app.netlify.com/sites/react-navigation-docs/deploys

**Comment by slorber** [comment]
Thanks @satya164 

Not sure why Netlify does not seem to use the build plugin but it seems we still see Webpack 5 persistent caching improvements:

image

**Comment by slorber** [comment]
ready for review!

File: package.json [hunk 1]
```diff
@@ -12,15 +12,16 @@
     "fetch-sponsors": "node scripts/fetch-sponsors.js"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-alpha.72",
-    "@docusaurus/plugin-google-analytics": "^2.0.0-alpha.66",
-    "@docusaurus/preset-classic": "^2.0.0-alpha.72",
+    "@docusaurus/core": "2.0.0-alpha.75",
+    "@docusaurus/plugin-google-analytics": "2.0.0-alpha.75",
```
Comment (eriveltonelias) [inline package.json:16]: Thanks @slorber for your PR! just one small thing to check before we land it.

@brentvatne could you check if this new version GA plugin still working the analytics?

Comment (slorber) [reply package.json:16]: Thanks

Do you expect me to do anything?

Comment (eriveltonelias) [reply package.json:16]: nope, all looks good to me @slorber, thanks again for your awesome work on docusarus.

Comment (brentvatne) [reply package.json:16]: @eriveltonelias - i can add you to the google analytics account, dm me on discord to let me know what email to add :)

**Pull request merged**